modules/yugabytedb/yugabytedb.go

```diff
@@ -0,0 +1,249 @@
+package yugabytedb
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+	"github.com/yugabyte/gocql"
+)
+
+const (
+	mastDashboardPort    = "7000"
+	tServerDashboardPort = "9000"
+)
+
+const (
+	ycqlPort = "9042/tcp"
+
+	ycqlKeyspaceEnv = "YCQL_KEYSPACE"
+	ycqlUserNameEnv = "YCQL_USER"
+	ycqlPasswordEnv = "YCQL_PASSWORD"
+
+	ycqlKeyspace = "yugabyte"
+	ycqlUserName = "yugabyte"
+	ycqlPassword = "yugabyte"
+)
+
+const (
+	ysqlPort = "5433/tcp"
+
+	ysqlDatabaseNameEnv     = "YSQL_DB"
+	ysqlDatabaseUserEnv     = "YSQL_USER"
+	ysqlDatabasePasswordEnv = "YSQL_PASSWORD"
+
+	ysqlDatabaseName     = "yugabyte"
+	ysqlDatabaseUser     = "yugabyte"
+	ysqlDatabasePassword = "yugabyte"
+)
+
+// YugabyteDBContainer represents the yugabyteDB container type used in the module
+type YugabyteDBContainer struct {
```

Comment:
suggestion: not sure if this is part of the generation template, if so we should consider changing that too in a separate PR, but this stutters in use e.g. `yugabytedb.YugabyteDBContainer` we should simplify to avoid that:
```suggestion
// Container represents the yugabyteDB container type used in the module.
type Container struct {
```

Reply:
This is indeed part of the auto genrated template. Though I agree with you something like `yugabytedb.Container` makes more sense and seems more idiomatic.

Reply:
Indeed. It's changed in v1: all modules will have a `Container` struct instead of an `XXXContainer`. Changing that would be a big BC that we should postpone for v1

Reply:
Alright I'll adjust this one to Container instead then and I'll open up a new PR to adjust the generator

Reply:
✅ Attended

Reply:
@mdelapenya as promised here's the PR https://github.com/testcontainers/testcontainers-go/pull/2831/files
I also fixed an issue related to missing imports